sdks/python/apache_beam/testing/analyzers/perf_analysis_test.py

```diff
@@ -193,6 +198,32 @@ def test_alert_on_data_with_reported_change_point(self, *args):
         big_query_metrics_fetcher=None)
     self.assertFalse(is_alert)
 
+  def test_change_point_has_anomaly_marker_in_gh_description(self):
+    metric_values, timestamps = get_fake_data_with_change_point()
+    timestamps = [datetime.datetime.fromtimestamp(ts) for ts in timestamps]
+    change_point_index = find_latest_change_point_index(metric_values)
+
+    description = github_issues_utils.get_issue_description(
+        test_name=self.test_id,
+        test_description=self.params['test_description'],
+        metric_name=self.params['metric_name'],
+        metric_values=metric_values,
+        timestamps=timestamps,
+        change_point_index=change_point_index,
+        max_results_to_display=(
+            constants._NUM_RESULTS_TO_DISPLAY_ON_ISSUE_DESCRIPTION))
+
+    runs_info = next((
```

Comment:
This is too specific to the formatting, and if one changes the formatting, one would have to modify the test. The "new lines joiner" constant a sample inconvenience that this creates. We could simplify this scenario by checking that anomaly marker occurs only once, and and that the string "metric_value: {anomaly_value} {_ANOMALY_MARKER}" occurs in the description.

Reply:
Changed it

Reply:
would it be sufficient to do pattern-matching in `description` variable without further processing in `runs_info`?

Reply:
No, the index of the runs_info would change if we add other things to description list.